### ISSUE TITLE
Unsupported keras layer type BatchNormalizationV1

### ISSUE THREAD

**Issue by victoriest** [deeplearning4j/deeplearning4j#8338]
#### Issue Description

Exception occured when load trained tensorflow.keras model(.h5)
```
Exception in thread "main" org.deeplearning4j.nn.modelimport.keras.exceptions.UnsupportedKerasConfigurationException: Unsupported keras layer type BatchNormalizationV1. Please file an issue at http://github.com/deeplearning4j/deeplearning4j/issues.
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasLayerUtils.getKerasLayerFromConfig(KerasLayerUtils.java:320)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.prepareLayers(KerasModel.java:218)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:164)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:96)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelBuilder.buildModel(KerasModelBuilder.java:307)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasModelAndWeights(KerasModelImport.java:172)
	at me.victoriest.saas.App.main(App.java:24)
```

#### Version Information

* Deeplearning4j 1.0.0-beta5
* Tensorflow 1.13.1
* jdk1.8
* Window10

#### Additional Information

Where applicable, please also provide:

* Full log or exception stack trace (ideally in a Gist: gist.github.com)
* pom.xml file or similar (also in a Gist)

**Comment by eraly** [comment]
Could you try using just vanilla keras? ie import keras and saving like that?

**Comment by victoriest** [comment]
Thanks for the reply. 
I retrained the model by using keras(not tf's). The issues seems been sloved cause the exception didn't shows again, but it still cant work:
```
Exception in thread "main" org.deeplearning4j.exception.DL4JInvalidConfigException: Invalid configuration for layer (idx=-1, name=max_pooling2d_2, type=SubsamplingLayer) for height dimension:  Invalid input configuration for kernel height. Require 0 < kH <= inHeight + 2*padH; got (kH=16, inHeight=14, padH=0)
```
It's another issuse, maybe. This one can be closed.

**Comment by AlexDBlack** [comment]
Thanks for the issue - for tf.keras, keep an eye on this issue: https://github.com/eclipse/deeplearning4j/issues/8348

As for the second issue - `Invalid configuration for layer (idx=-1, name=max_pooling2d_2, type=SubsamplingLayer) for height dimension:  Invalid input configuration for kernel height. Require 0 < kH <= inHeight + 2*padH; got (kH=16, inHeight=14, padH=0)`:
That's an unusual kernel size, 16x16 (most CNNs these days are like 2x2 or 3x3 kernels). Unless it's being used as a type of global pooling layer...
We have seen an issue in deconv layers where Keras implicitly adds padding for kernel > input size cases, it may be a similar issue here.
Feel free to open another issue for that, and we'll take a look (we need more details though, model configuration / file to reproduce the problem). Thanks.
